### DISCUSSION TITLE
one to many relation

### DISCUSSION THREAD

**Discussion by BigGitWorld** [simolus3/drift#1491] [General]
Hi.
Where can I find a complete one-to-many relation example?

**Comment by simolus3** [comment]
There's an example app [here](https://github.com/rodydavis/moor_shared/blob/c23c4139759b570a77b91b1fa585c787fa38d70e/lib/src/database/database.dart#L11-L29). It's a todo app with categories, where each category can hold many todo items but each todo item can be at most in one category.
